docs/serving/configuring-autoscaling.md

```diff
@@ -33,7 +33,6 @@ data:
  container-concurrency-target-default: 100
  container-concurrency-target-percentage: 1.0
  enable-scale-to-zero: true
```

Comment:
This seems to be missing a few other options that are in https://github.com/knative/serving/blob/master/config/core/configmaps/autoscaler.yaml

Like scale-to-zero-grace-period, enable-graceful-shutdown, and requests-per-second-target-default. Can you add those as well with this change?

Reply:
Hm, I didn't want this to be an overhaul of the docs :)
Also `enable-graceful-shutdown` is not working yet.

Reply:
scale-to-zero-grace period is described below.